lib/cache.js

```diff
@@ -0,0 +1,27 @@
+const NodeCache = require('node-cache');
+
+const rateLimitCache = new NodeCache({ stdTTL: 20, checkperiod: 60 });
```

Comment:
do we really need the stdTTL to be changed? 

we are setting TTL manually every time, so it doesn't have any effect anyway

Reply:
Will remove it.